tonic-build/src/client.rs

```diff
@@ -211,15 +220,19 @@ fn generate_methods<T: Service>(
     stream
 }
 
-fn generate_unary<T: Method>(
-    method: &T,
+fn generate_unary<T: Service>(
+    service: &T,
+    method: &<T as Service>::Method,
```

Comment:
```suggestion
    method: &T::Method,
```

This should work no?